### ISSUE TITLE
Layered ban and admin lists

### ISSUE THREAD

**Issue by ForNeVeR** [Liminiens/grinder#15]
We need to include layers of banlists.

Consider we have three chats: `chat1` and `chat2` belong to the main network, and `chat3` is a side chat.

We need to change Grinder so `chat3` is semi-isolated from the main network: admins of the main network are also admins in `chat3`, but `chat3` admins aren't admins in `chat1` and `chat2`. People banned in `chat1` and `chat2` are also automatically banned in `chat3`, but not the other way around.

In other words, `chat3` should automatically receive admins and bans from the main network, but the main network shouldn't receive admins and bans from the `chat3`.
